src/Illuminate/Foundation/Bootstrap/DetectEnvironment.php

```diff
@@ -24,4 +26,21 @@ public function bootstrap(Application $app)
             }
         }
     }
+
+    /**
+     * Detect if a custom env file matching the APP_ENV exists.
+     *
+     * @param  Illuminate\Contracts\Foundation\Application  $app
+     * @return mixed
```

Comment:
We are not making using the return of the function at all, so why not just `void`:
```php
$fileName = $app->environmentFile().'.'.env('APP_ENV');

if (file_exists($app->environmentPath().'/'.$fileName)) {
    $app->loadEnvironmentFrom($fileName);
}
```
?